lib/cdo/aws/cloudfront.rb

```diff
@@ -135,6 +135,19 @@ def self.distribution_config(app, origin, aliases, ssl_cert=nil)
       # Add root-domain aliases to production environment stack.
       aliases += CONFIG[app][:aliases] if rack_env?(:production)
 
+      origin_custom_headers = [
+        {
+          HeaderName: 'X-Cdo-INF506',
```

Comment:
Can we choose a header name that explains what its purpose is? `x-Cdo-Auth-CloudFront`? Or `x-Cdo-Auth-Fowarded-By-CloudFront`?